### ISSUE TITLE
Documentation: published service attribute external_id -> external_ip

### ISSUE THREAD

**Issue by ja6a** [skytap/terraform-provider-skytap#4]
Documentation reads `external_id` for an attribute on published service, describing it as an external ID, it is, in fact `external_ip` representing the published service's external IP.

**Comment by jchenry** [comment]
I am concerned about using the internal IP as the key, since technically, this could be dynamically assigned and not specified. 

ideally,  an interpolation to get to the attributes would be in a more direct way, i.e. 
`skytap_vm.my_vm.network_interface.eth0.published_service.ssh`

and expressed as a resource: 
```hcl 
resource "skytap_vm" "vm2" {
   network_interface "eth0" {
 ...
    published_service "ssh" {
      "internal_port" = 22
    }
  }
```


I'm unsure if this is acheivable. I'm sure other approaches would work as well.

**Comment by ja6a** [comment]
> I am concerned about using the internal IP as the key, since technically, this could be dynamically assigned and not specified.
> 
> ideally, an interpolation to get to the attributes would be in a more direct way, i.e.
> `skytap_vm.my_vm.network_interface.eth0.published_service.ssh`
> 
> and expressed as a resource:
> 
> ```terraform
> resource "skytap_vm" "vm2" {
>    network_interface "eth0" {
>  ...
>     published_service "ssh" {
>       "internal_port" = 22
>     }
>   }
> ```
> 
> I'm unsure if this is acheivable. I'm sure other approaches would work as well.

I think this comment belongs here: #7

**Comment by ja6a** [comment]
Resolved in PR #8
